### PR TITLE
added turkish language

### PR DESCRIPTION
**Pull request by rizaergun**

### CODE REVIEW

**Review by bndw — CHANGES_REQUESTED** [review]
Hi @rizaergun and thanks for the contribution! Would you mind running `make fmt.write` to format the source code changes?

Also, the README has a supported language table at the bottom -- would you mind adding a row for Turkish?

**Comment by rizaergun** [comment]
> Hi @rizaergun and thanks for the contribution! Would you mind running `make fmt.write` to format the source code changes?
> 
> Also, the README has a supported language table at the bottom -- would you mind adding a row for Turkish?

hi @bndw i did them.

**Comment by bndw** [comment]
@rizaergun Did you see the existing PR adding Turkish https://github.com/bndw/wifi-card/pull/105 ?

**Comment by rizaergun** [comment]
@bndw yeah I saw this right now and I think there is some problems and it is open for 6 days.

File: README.md [hunk 1]
```diff
@@ -51,4 +51,5 @@ make fmt.write
 | Russian    | [Teraskull](https://github.com/Teraskull)     |
 | Ukrainian  | [Teraskull](https://github.com/Teraskull)     |
 | Dutch      | [wouterbrink](https://github.com/wouterbrink) |
-| French     | [Divlo](https://github.com/Divlo)
+| French     | [Divlo](https://github.com/Divlo)             |
+| Turkish    | [Riza Ergun](https://github.com/rizaergun)
```
Comment (bndw) [inline README.md:55]: Missing the trailing `|`

File: src/App.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ function App() {
           <option value="nl-NL">Nederlands</option>
           <option value="fr-FR">French - FR (Français)</option>
           <option value="pt-BR">pt-BR</option>
+          <option value="tr-TR">Türkçe</option>
```
Comment (bndw) [inline src/App.js:34]: To help out the effort in https://github.com/bndw/wifi-card/pull/81, this should read `Turkish (Türkçe)`

**Review by bndw — CHANGES_REQUESTED** [review]
Couple small changes and we're good to go!

**Comment by rizaergun** [comment]
@bndw i fixed all of them.

**Review by bndw — APPROVED** [review]

**Pull request merged**